TestSpec is not really flaky

See #237 

You can't run the tests without cloning `envelope-spec` into the test directory:

```
=== RUN   TestSpec
    spec_test.go:29:
                Error Trace:    /home/go-ssb/go-ssb/private/box2/spec_test.go:29
                Error:          Received unexpected error:
                                open spec/vectors: no such file or directory
                Test:           TestSpec
                Messages:       open vectors dir
    spec_test.go:30: suggestion: run 'git clone https://github.com/ssbc/envelope-spec spec'
    spec_test.go:31: or clone it somewhere else and create a symlink to it here named 'spec'
--- FAIL: TestSpec (0.00s)
FAIL
FAIL    github.com/ssbc/go-ssb/private/box2     0.008s
FAIL
```

Not sure if there's a way to integrate this into the test suite or if the CI system downloads this when testing.  Maybe this is why it was failing before?  I ran it about 150 times and had zero failures once I downloaded `envelope-spec`.

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,9 @@ endif
 
 .PHONY: test
 test:
+	test -d private/box2/spec || git clone https://github.com/ssbc/envelope-spec private/box2/spec
 	$(foreach pkg, $(PKGS), echo -en "\n        $(pkg)\r"; LIBRARIAN_WRITEALL=0 go test $(TESTFLAGS) $(pkg) || exit 1;)
+	rm -rf private/box2/spec
 
 .PHONY: racetest
 racetest:

--- a/private/box2/spec_test.go
+++ b/private/box2/spec_test.go
@@ -14,7 +14,6 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/ssbc/go-ssb/internal/testutils"
 )
 
 func seq(start, end int) []byte {
@@ -26,10 +25,6 @@ func seq(start, end int) []byte {
 }
 
 func TestSpec(t *testing.T) {
-	if testutils.SkipOnCI(t) {
-		return
-	}
-
 	dir, err := os.Open(filepath.Join("spec", "vectors"))
 	if !assert.NoError(t, err, "open vectors dir") {
 		t.Log("suggestion: run 'git clone https://github.com/ssbc/envelope-spec spec'")

